src/physics/arcade/Body.js

```diff
@@ -438,6 +438,21 @@ var Body = new Class({
         this.maxVelocity = new Vector2(10000, 10000);
 
         /**
+         * The Body's maximum speed.
+         * 
+         * This limits Body's scalar value of speed. Always has positive value.
+         * 
+         * Negative value means that there is no limitation on speed.
+         * 
+         * @name Phaser.Physics.Arcade.Body#maxSpeed
+         * @type {number}
+         * @since 3.13.0 
+         */
+        this.maxSpeed = -1;
+
+        /**
+         * If this Body is `immovable` and in motion, this the proportion of this Body's movement received by the riding body on each axis, relative to 1.
+         * The default value (1, 0) moves the riding body horizontally in equal proportion and vertically not at all.
```

Comment:
This comment part looks wrong to me? Is it intended to replace the first two lines? Or did the base comment change and this PR is accidentally reverting/prepending it back?